lib/operations/arithmetic/substract.js

```diff
@@ -0,0 +1,10 @@
+function substract(a, b) {
+  if (b === undefined) {
+    return -a;
+  }
+
+  return a - b;
+}
+
+substract.code = '-';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`-` is an existing operator